src/lib/config.ts

```diff
@@ -0,0 +1,15 @@
+export function getK8sVersionEOLDate(version: string) {
+  if (!version) return null;
+
+  const k8sVersions = window.config.info.general.kubernetesVersions;
+  if (!k8sVersions) return null;
+
+  const versionParts = version.split('.');
+  if (versionParts.length < 2) return null;
+  const minor = `${versionParts[0]}.${versionParts[1]}`;
+
+  const versionInfo = k8sVersions[minor];
```

Comment:
Will it work when it's constructed like below?

```
                  '{!= version !}': {
                    minorVersion: '{!= versionInfo.minorVersion !}',
                    eolDate: '{!= versionInfo.eolDate !}',
                  },             
```

Shouldn't it be `k8sVersions[version]`?

Reply:
`version` is actually the minor version here
https://github.com/giantswarm/config/blob/main/include/k8s-versions.yaml.template

But you're right, there's nothing enforcing that, I should change the template to always have the minor version as the key